src/test/java/eu/europeana/metis/sandbox/service/dataset/DatasetReportServiceImplTest.java

```diff
@@ -72,22 +73,31 @@ void getReportWithErrors_expectSuccess() {
         new DatasetInfoDto("","", LocalDateTime.now(), Language.NL, Country.NETHERLANDS,
             false, false));
 
```

Comment:
as a suggestion, you can remove duplicated code, the code will be shorter.
```
RecordEntity recordEntity1 = new RecordEntity("europeanaId1", "providerId1", "1");
recordEntity1.setId(1L);
```

an idea of the function. 

```
private static RecordEntity getTestRecordEntity(final Long recordId) {
    RecordEntity recordEntity = new RecordEntity("europeanaId"+recordId.toString(), "providerId"+recordId.toString(), recordId.toString());
    recordEntity.setId(recordId);
    return recordEntity;
}
```
so the following code will look like
```
var errorView1 = new ErrorLogViewImpl(1L, getTestRecordEntity(1L), Step.VALIDATE_EXTERNAL, Status.FAIL,
        "cvc-complex-type.4: Attribute 'resource' must appear on element 'edm:object'.");
```

Reply:
That is a good suggestion, I will add these, thanks!